src/httpParsing/parseHar.ts

```diff
@@ -11,7 +11,27 @@ const getEntries = (harFile: Record<string, any>) => {
   return entries;
 };
 
+const parseContent = (input: {
+  mimeType: string;
+  text: string;
+}): har.postData | undefined => {
+  if (!input) {
+    return undefined;
+  }
+  if (input.mimeType !== "application/json") {
+    console.warn("Only application/json is supported!");
+    return undefined;
+  }
+  return {
+    mimeType: input.mimeType,
+    text: input.text,
+    parsed: JSON.parse(input.text),
```

Comment:
I think parsing the content here will 1) be memory-intensive and 2) make adding additional mime types tricky. I think we should keep the string and only parse it if and when we validate it.

Reply:
agreed, will change